tools/sgen/input/FluentUIStyle.yml

```diff
@@ -44,6 +44,23 @@ Colors:
     - brandSelected: "FluentUIColor(light: $Colors.Brand.shade20, dark: $Colors.Brand.shade20)"
     - brandDisabled: "FluentUIColor(light: NamedColor(grey74), dark: NamedColor(grey36))"
 
+  Notification:
+    - primaryToastForeground: "FluentUIColor(light: $Colors.Brand.shade10, dark: $Colors.Brand.shade30)"
+    - primaryBarBackground: "FluentUIColor(light: $Colors.Brand.tint40, dark: $Colors.Brand.tint10)"
+    - primaryBarForeground: "FluentUIColor(light: $Colors.Brand.shade20, dark: NamedColor(black))"
+    - primaryOutlineBarBackground: "FluentUIColor(light: $Colors.Brand.primary, dark: #F7F7F7)"
```

Comment:
I'd expect the backgrounds that are using grayScale colors to be coming from the Background definition:

  ```
Background:
    - neutral1: "FluentUIColor(light: NamedColor(white), dark: NamedColor(black), darkElevated: NamedColor(grey4))"
    - neutral2: "FluentUIColor(light: NamedColor(grey98), dark: NamedColor(grey4), darkElevated: NamedColor(grey8))"
    - neutral3: "FluentUIColor(light: NamedColor(grey96), dark: NamedColor(grey8), darkElevated: NamedColor(grey12))"
    - neutral4: "FluentUIColor(light: NamedColor(grey94), dark: NamedColor(grey12), darkElevated: NamedColor(grey16))"
    - neutral5: "FluentUIColor(light: NamedColor(grey92), dark: NamedColor(grey36))"
    - neutralDisabled: "FluentUIColor(light: NamedColor(grey88), dark: NamedColor(grey84))"
    - brandRest: "FluentUIColor(light: $Colors.Brand.primary, dark: $Colors.Brand.primary)"
    - brandHover: "FluentUIColor(light: $Colors.Brand.shade10, dark: $Colors.Brand.shade10)"
    - brandPressed: "FluentUIColor(light: $Colors.Brand.shade30, dark: $Colors.Brand.shade30)"
    - brandSelected: "FluentUIColor(light: $Colors.Brand.shade20, dark: $Colors.Brand.shade20)"
    - brandDisabled: "FluentUIColor(light: NamedColor(grey88), dark: NamedColor(grey84))"
    - surfacePrimary: "FluentUIColor(light: NamedColor(white), dark: NamedColor(black), darkElevated: NamedColor(grey8))"
    - surfaceQuaternary: "FluentUIColor(light: NamedColor(grey88), dark: NamedColor(grey26))"
```